sormas-api/src/main/resources/captions.properties

```diff
@@ -340,6 +340,7 @@ CaseData.smallpoxVaccinationReceived=Was a Smallpox vaccination received in the
 CaseData.smallpoxVaccinationScar=Is a Smallpox vaccination scar present?
 CaseData.surveillanceOfficer=Responsible surveillance officer
 CaseData.symptoms=Symptoms
+CaseData.symptomsOnsetDate=Symptoms onset date
```

Comment:
I'd like you to remove this property and use the already existing `Symptoms.onsetDate` property instead for the column caption (see my comment in CaseGridDetailed.java)